pkg/settings/cluster/cockroach_versions.go

```diff
@@ -283,6 +285,12 @@ var versionsSingleton = keyedVersions([]keyedVersion{
 		Key:     VersionLoadBasedRebalancing,
 		Version: roachpb.Version{Major: 2, Minor: 0, Unstable: 14},
 	},
+	// Version2_1 should go here.
```

Comment:
I'm not 100% sure how this version stuff works. Do we need to wait until a 2.1.0 is made before making a 2.1.1?

Reply:
Yeah, things get a little weird if we do this out of order. Today's the day to make the 2.1 cluster version anyway. 

But I don't think it's a great idea to tie a change like this to the cluster version - the change happens at exactly the point where the upgrade becomes irreversible. I think it might be better to just make the change as soon as the new binary is used while rolling back is still possible.